content/en/blog/2023/submitting-your-first-conference-talk.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Submitting Your First Conference Talk
+linkTitle: Submitting a CFP
+date: 2023-02-07
+author: '[Josh Lee](https://github.com/joshleecreates/) (IBM)'
+canonical_url: https://joshuamlee.com/submitting-your-first-conference-talk/
+---
+
+Submitting a conference talk in the tech industry can be a thrilling and
+nerve-wracking experience, especially if it’s your first time. The thought of
+presenting your ideas to a room full of experts and industry professionals can
+be intimidating, but also an opportunity to showcase your skills, gain
+recognition, and network with others in your field. On Wednesday, February 1st,
+2023, the
+[OpenTelemetry End User Working Group](/community/end-user/)
+ran an Observability Abstract Workshop. The goal was to help folks who are new
+to speaking, and are interested in submitting an observability panel or talk.
+The workshop featured advice and expertise from OpenTelemetry community members
+[Reese Lee](https://github.com/reese-lee),
+[Daniel Kim](https://github.com/lazyplatypus),
+[Rynn Mancuso](https://github.com/musingvirtual),
+[Juraci Paixão Kröhling](https://github.com/jpkrohling), and
+[Adriana Villela](https://github.com/avillela). What follows is a summary of the
+workshop's main takeaways. Whether you’re a seasoned speaker or just starting
+out, this guide will provide you with tips to help you make the most of your
+conference talk submission and increase your chances of being selected to speak.
```

Comment:
I know you just made this change -- I recommend breaking this up into at least 2 paragraphs, maybe even 3, for better readability, e.g.:

"Submitting..."

"On Wednesday..."

"What follows..."